classes/class_demo1.py

```diff
@@ -0,0 +1,14 @@
+"""
+Object Oriented Programming
+Everything in Python is an object (ex: car, animal)
+Objects have properties
+ex: car
+    attributes/properties: colour, year, model, make (static things)
+    methods: start, stop, break (action is happening)
+"""
+
+s = "This is a string"
+print(str(s.upper()))
```

Comment:
You don't have to convert it with str() if ou print only one object. Especially if you expect for that object to be a str object. I think it's redundant.